src/supermarket/config/initializers/paperclip.rb

```diff
@@ -21,7 +17,8 @@
       },
       path: path,
       bucket: ENV['S3_BUCKET'],
-      s3_protocol: ENV['PROTOCOL']
+      s3_protocol: ENV['PROTOCOL'],
+      s3_region: ENV['S3_REGION']
     }
 
     if ENV['S3_PRIVATE_OBJECTS'].present?
```

Comment:
Tangentially, I've noticed these `.present?` checks which make me sad. I submit that they should be some sort of truthy check on the value instead of just having the variable present in the environment.

The following is probably going to confuse a human at some point.
```
> export S3_PRIVATE_OBJECTS="false"

> be rails c
Loading development environment (Rails 4.2.7.1)
[1] pry(main)> ENV['S3_PRIVATE_OBJECTS']
=> "false"
[2] pry(main)> ENV['S3_PRIVATE_OBJECTS'].present?
=> true
```